crates/opencascade-sys/src/lib.rs

```diff
@@ -179,6 +187,10 @@ pub mod ffi {
         pub fn IsNull(self: &TopoDS_Shape) -> bool;
         pub fn IsEqual(self: &TopoDS_Shape, other: &TopoDS_Shape) -> bool;
 
+        type TopAbs_Orientation;
+        pub fn Orientation(self: &TopoDS_Shape) -> TopAbs_Orientation;
+        pub fn Orientation(self: &TopoDS_Face) -> TopAbs_Orientation;
```

Comment:
Are these currently used or needed?

Reply:
They are needed. Its not used in the test but if you want to calculate the normals for each triangle you need this info,